### PR TITLE
Configurable macro for static

### PR DESCRIPTION
**Pull request by nZac**

### CODE REVIEW

File: keg_elements/tests/test_templates.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+
+import keg
+from keg_elements.forms import Form
+from pyquery import PyQuery
+from wtforms import StringField
+
+
+class TestTemplates(object):
```
Comment (rsyring) [inline keg_elements/tests/test_templates.py]: YAY TESTS!!!!!!!!!!!  Seriously, I'm smiling!

Comment (3noch) [inline keg_elements/tests/test_templates.py]: :smile: I got a little stuck, so I committed (and apparently @nZac created a PR??) in the hopes that I could get some help soon.

**Comment by 3noch** [comment]
@rsyring @nZac This now has working tests.

**Comment by 3noch** [comment]
The goal of this PR is to allow static fields to have custom rendering for the value. I'm using this right now to change how we display sensitive information when the current user is missing certain permissions. This change allows me to quickly inject a custom rendering macro for the field's value. It *may* be possible to extend this to work with dynamic fields as well, but that seems much more complicated.

**Comment by nZac** [comment]
I am cool with this. :+1:

File: kegel_app/templates/section-macro.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{% import 'keg_elements/forms/horizontal.html' as dynamic_render %}
+{% import 'keg_elements/forms/horizontal_static.html' as static_render %}
+
+{%- macro doubled(value) -%}
```
Comment (rsyring) [inline kegel_app/templates/section-macro.html]: Is this macro needed in this file?

**Comment by rsyring** [comment]
I'm good with a merge, thanks for getting tests going!

File: kegel_app/templates/section-macro.html (hunk 2 above)
Comment (3noch) [inline kegel_app/templates/section-macro.html]: Not in the slightest! ~~It's gone.~~~

**Comment by nZac** [comment]
@rsyring You want me to take care of the merge?

**Comment by 3noch** [comment]
My pleasure! Thanks for your help with getting them running. That's why I didn't add them earlier... :/

**Pull request merged**